### DISCUSSION TITLE
How To Open A Web App

### DISCUSSION THREAD

**Discussion by douglasamcintosh** [ScoopInstaller/Scoop#5147] [Help and Questions] [answered]
Hello,

New to scoop. Trying to set this up

https://github.com/marcrobledo/RomPatcher.js/

I want to set up a shortcut that will open the app (effectively a web page) in the systems default browser (or any browser will do). I've had a look and can't see an obvious solution.

Thanks,

Douglas

**Comment by rashil2000** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/46838874/190271810-a15f0e67-fac5-4c4c-a937-04d416b91755.png)

Go to the website https://www.marcrobledo.com/RomPatcher.js/ using any browser

Click on the checkered-box icon as shown above

It will install the webapp

**Reply by douglasamcintosh** [reply]
Hi. Thanks for the response. I'm trying to make an easy link so I can install it via scoop and then open the web app from the menu.

**Reply by rashil2000** [reply]
Webapps cannot be installed via Scoop

**Reply by douglasamcintosh** [reply]
That's what I feared. Thanks for the response.
